### ISSUE TITLE
Gradle not using custom repository during gradle init

### ISSUE THREAD

**Issue by motoroler** [gradle/gradle#22061]
```
------------------------------------------------------------
Gradle 7.4
------------------------------------------------------------

Build time:   2022-02-08 09:58:38 UTC
Revision:     f0d9291c04b90b59445041eaa75b2ee744162586

Kotlin:       1.5.31
Groovy:       3.0.9
Ant:          Apache Ant(TM) version 1.10.11 compiled on July 10 2021
JVM:          11 (Oracle Corporation 11+28)
OS:           Windows 10 10.0 amd64
------------------------------------------------------------
```
I am trying to migrate my project from maven to gradle, but I can't access the custom company repository. Gradle connects to the central repository by default, and disregards both my maven settings.xml and init.gradle. The issue is the same as https://github.com/gradle/gradle/issues/19884 but I am using a windows machine.

### Expected Behavior
Gradle should use the repository settings from init.gradle.

### Current Behavior
Gradle ignores the settings and tries to access https://repo.maven.apache.org/maven2 instead and gets blocked by corporate proxy. Same settings work fine while running any other gradle command, such as build.

### Context
I can not convert a maven project to gradle while behind a corporate proxy that blocks maven repos.

### Steps to Reproduce (for bugs)
1- Be behind a proxy that blocks maven repos.
2- Have a maven project
3- Open maven project folder and run gradle init with init.gradle config.

### Your Environment
init.gradle file:

    allprojects {

        repositories {

            mavenLocal()

	    maven { url "custom-repo1" }

	    maven { url "custom-repo2" }

	    maven { url "custom-repo3" }
    }
}
Settings xml:

<mirrors>
	<mirror>
		<mirrorOf>*</mirrorOf>
		<name>remote-repos</name>
		<url>custom-repo</url>
		<id>remote-repos</id>
	</mirror>

	<mirror>
		<mirrorOf>central</mirrorOf>
		<name>apache-repo</name>
		<url>custom-repo</url>
		<id>apache-repo</id>
	</mirror>
</mirrors>

<repositories>
			<repository>
				<snapshots>
					<enabled>false</enabled>
				</snapshots>
				<id>central</id>
				<name>libs-release</name>
				<url>custom-repo</url>
			</repository>
			<repository>
				<snapshots />
				<id>snapshots</id>
				<name>libs-snapshot</name>
				<url>custom-repo</url>
			</repository>
		</repositories>
Build scan URL:
I ran gradle init --scan and got the following error:

FAILURE: Build failed with an exception.

Where:
Auto-applied by using --scan

What went wrong:
Plugin [id: 'com.gradle.enterprise', version: '3.8.1', artifact: 'com.gradle:gradle-enterprise-gradle-plugin:3.8.1'] was not found in any of the following sources:

Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
Plugin Repositories (could not resolve plugin artifact 'com.gradle:gradle-enterprise-gradle-plugin:3.8.1')
Searched in the following repositories:
Gradle Central Plugin Repository
Try:
Run with --stacktrace option to get the stack trace.
Run with --info or --debug option to get more log output.
Run with --scan to get full insights.

Get more help at https://help.gradle.org/
This is probably because it did not use my repo settings and got blocked by proxy.

**Comment by jbartok** [comment]
Thank you for your interest in Gradle!

Your issue appears to be a question about how to use Gradle. Your question might be better answered on our forums or StackOverflow. We have several resources on [help.gradle.org](https://help.gradle.org) that you should check out first.

If you think you have come across a Gradle bug, please file a new issue describing the problem and how to reproduce it.  

If you think Gradle is missing a feature, please file a new issue describing your use case for the feature.

**Comment by samyack15** [comment]
I'm have this error can someone help. I'm new to Android development

FAILURE: Build failed with an exception.

* Where:
Auto-applied by using --scan

* What went wrong:
Plugin [id: 'com.gradle.enterprise', version: '3.8.1', artifact: 'com.gradle:gradle-enterprise-gradle-plugin:3.8.1'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (could not resolve plugin artifact 'com.gradle:gradle-enterprise-gradle-plugin:3.8.1')
  Searched in the following repositories:
    Gradle Central Plugin Repository
    Google
    MavenRepo
